draft-ietf-oauth-device-flow.xml

```diff
@@ -921,8 +933,8 @@
       <t>
         Roshni Chandrashekhar, Marius Scurtescu, Breno de Medeiros,
         Stein Myrseth, Simon Moffatt, Brian Campbell, James Manger,
-        Justin Richer, Ken Wang, Steven E.&nbsp;Wright, Nat Sakimura, and
-        Torsten Lodderstedt.
+        Justin Richer, Ken Wang, Steven E.&nbsp;Wright, Nat Sakimura,
+        Torsten Lodderstedt, Eric Fazendin, and Andrew Sciberras.
```

Comment:
Please alphabetize this list.

Reply:
Done. I also added a credit for our chairs and ADs, and slightly clarified the credit for the earlier contributions.